hosts/testagent/configuration.nix

```diff
@@ -75,6 +77,36 @@ in {
   # it only installs java and sets up jenkins user
   services.jenkinsSlave.enable = true;
 
+  # Gives jenkins user sudo rights without password and serial connection rights
+  users.users.jenkins.extraGroups = [ "wheel" "dialout" "tty"];
+
+  # Open connection to Jenkins controller as a systemd service
+  systemd.services.jenkins-connection = {
+    after = ["network.target"];
+    wantedBy = ["multi-user.target"];
+    path = [
+    pkgs.jdk
+    pkgs.git
+    pkgs.bashInteractive
+    pkgs.coreutils
+    pkgs.util-linux
+    pkgs.nix
+    pkgs.zstd
+    pkgs.jq
+    pkgs.csvkit
+    pkgs.sudo
+    brainstem
+    inputs.robot-framework.packages.${pkgs.system}.ghaf-robot
+    ];
+    serviceConfig = {
+      Type = "simple";
+      User = "vjuntunen";
+      WorkingDirectory = "/home/vjuntunen/Jenkins-agent/";
```

Comment:
Is this really the user and working directory we want to run this? Why?

Reply:
Well for now it is and I have different task to change it to Jenkins user and maybe better to do that with own PR?. Like we have talked in our meetings.